src/core/opamStd.ml

```diff
@@ -995,22 +995,9 @@ module OpamSys = struct
 
   let etc () = "/etc"
 
-  let memo_command =
```

Comment:
Is there a reason to remove memoisation for remaining uname cmd calls ?

Reply:
the rest of the commands are called exactly at most once per opam calls so there is no reason to have a memorization function around